video/image_writer.c

```diff
@@ -416,21 +699,27 @@ static struct mp_image *convert_image(struct mp_image *image, int destfmt,
 
 bool write_image(struct mp_image *image, const struct image_writer_opts *opts,
                 const char *filename, struct mpv_global *global,
-                 struct mp_log *log)
+                struct mp_log *log)
```

Comment:
Nitpicking here but actually could you instead *add* a space to the `const char *filename, struct mpv_global *global,` line? It should all be lined up under `struct mp_image *image, const struct image_writer_opts *opts`.

Reply:
done.